src/app/components/elements/PageTitle.tsx

```diff
@@ -47,6 +48,8 @@ export interface PageTitleProps {
   help?: ReactElement;
   className?: string;
   audienceViews?: ViewingContext[];
+  editable?: boolean;
+  onEdit?: (argo0: string) => void;
```

Comment:
Could you rename this function argument to something a bit more meaningful, please? Maybe something like `newPageTitle` or something like that. We have the same issue in `EditablePageTitle`.